### PR TITLE
Lgd consistency

### PR DESCRIPTION
**Pull request by MirijaHa**

### CODE REVIEW

File: vignettes/articles/05-read-the-outputs.Rmd [hunk 1]
```diff
@@ -349,12 +346,7 @@ stress testing model. The information on all model input parameters [is describe
 the following set of input parameters:
 
 * `asset_type_arg` is the type of the analysed financial asset in the portfolio. 
-* `lgd_senior_claims_arg` is the loss given default used in the expected loss
-calculation of loans for senior claims (notably for corporate loans, so this is
-a potentially important parameter in this calculation)
-* `lgd_subordinated_claims_arg` is the loss given default used in the expected 
-loss calculation of loans for subordinated claims (notably for corporate bonds.
-Not of relevance in the loans calculation)
+* `lgd__arg` is the loss given default used in the expected loss calculation.
```
Comment (jacobvjk) [inline vignettes/articles/05-read-the-outputs.Rmd:349]: typo, please remove one underscore

File: R/run_stress_test.R [hunk 2]
```diff
@@ -84,8 +80,7 @@ run_stress_test <- function(asset_type,
     baseline_scenario = baseline_scenario,
     shock_scenario = shock_scenario,
     scenario_geography = scenario_geography,
-    lgd_senior_claims = lgd_senior_claims,
```
Comment (jacobvjk) [inline R/run_stress_test.R:87]: after merging master, the equivalent adjustments are needed for `run_lrisk()`

**Review by jacobvjk — CHANGES_REQUESTED** [review]
see comments

File: vignettes/articles/05-read-the-outputs.Rmd (hunk 1 above)
Comment (MirijaHa) [reply vignettes/articles/05-read-the-outputs.Rmd:349]: done

File: R/run_stress_test.R (hunk 2 above)
Comment (MirijaHa) [reply R/run_stress_test.R:87]: done

**Review by jacobvjk — APPROVED** [review]
lgtm

**Pull request merged**